crates/uv/tests/it/common/mod.rs

```diff
@@ -457,6 +467,24 @@ impl TestContext {
         command
     }
 
+    fn disallow_git_cli(bin_dir: &Path) -> std::io::Result<()> {
+        let contents = r"#!/bin/sh
+    echo 'error: `git` operations are not allowed — are you missing a cfg for the `git` feature?' >&2
+    exit 127";
+        let git = bin_dir.join(format!("git{}", env::consts::EXE_SUFFIX));
+        fs_err::write(&git, contents)?;
```

Comment:
It's funny, this is all *kinds* of weird on windows but that's *fine* because we just need windows to explode (especially because we only need this to break on *some* platform so the CI throws a fit and tells you to add a cfg).

Reply:
Yeah haha it doesn't fail with a nice error message on Windows but it does fail loudly..

Reply:
Is there something better we could do that remains trivial?

Reply:
I don't think so, especially if anything specifically looks for `git.exe` (windows PATH resolution is weird and magic about suffixes and there's two different APIs for looking up things on PATH, it's a whole thing).

Reply:
(Notably the thing JS does to shim binaries onto path doesn't work with rust's Command because of all this, blech)